src/components/Button/Button.styles.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, styled } from '@mui/material';
+
+export const StyledButton = styled(Button)(({ theme }) => ({
```

Comment:
We can destructure if we need to use multiple values from theme
```suggestion
export const StyledButton = styled(Button)(({ theme }) => ({
   const { palette, shape, spacing } = theme;
```